scripts/pinata-upload.ts

```diff
@@ -15,12 +15,29 @@ const pinata = pinataSDK(apiKey, secretKey);
 export async function pinataUpload() {
   // Config
   console.log(
-    "Deploying files to IPFS via Pinata using the following configuration:"
+    'Deploying files to IPFS via Pinata using the following configuration:'
   );
   console.log(config);
 
-  const imagesBasePath = path.join(__dirname, "../images");
-  const metadataBasePath = path.join(__dirname, "../metadata");
+  // Upload collection showcase image + metadata
+  const readableStreamForFile = fs.createReadStream(config.image);
+  const result = await pinata.pinFileToIPFS(readableStreamForFile);
+
+  // Create collection metadata
+  // https://docs.opensea.io/docs/contract-level-metadata
+  const collectionMetadata = {
+    name: config.name,
+    description: config.description,
+    image: `ipfs://${result.IpfsHash}`,
+    seller_fee_basis_points: config.royaltyShare * 100,
```

Comment:
So royalties would live in two places with this right? This is a bit worrisome because it introduces a syncing issue (complects two braids lol). i.e: What if sg721 introduces a way for royalties to be updated? Also its easy to put two different values in both places.

I say either we either remove royalties from sg721 or remove this and add a warning saying not to add it (and if it exists, then the contract royalties will take precedence).

Reply:
Can remove it here. Was just going off of the open sea metadata standards.

Reply:
Removed.